maven-plugin-api/src/main/java/org/apache/maven/plugin/descriptor/MojoDescriptor.java

```diff
@@ -170,15 +170,15 @@ public void setLanguage( String language )
     }
 
     /**
-     * @return <code>true</code> if the Mojo is deprecated, <code>false</code> otherwise.
+     * @return Description with reason of a Mojo deprecation.
      */
     public String getDeprecated()
     {
         return deprecated;
     }
 
     /**
-     * @param deprecated <code>true</code> to deprecate the Mojo, <code>false</code> otherwise.
+     * @param deprecated Set description with reason of a Mojo deprecation.
```

Comment:
The parameter does not set, the method does. So the param desc should ge identical with `#getDeprecated()` return value.